src/components/Cluster/ClusterDetail/__tests__/UpgradeClusterModal.tsx

```diff
@@ -39,13 +39,13 @@ function createRelease(version: string, active: boolean): IRelease {
 function createInitialState(
   releases: Record<string, IRelease>,
   isAdmin: boolean = false
-) {
+): Partial<IState> {
   return {
-    entities: {
+    entities: ({
       releases: {
         items: releases,
       },
-    },
+    } as unknown) as IState['entities'],
```

Comment:
Temporary, since there is no `releases` property in the `IState` interface, but not in the scope of this PR